Makefile

```diff
@@ -19,275 +23,45 @@ SHELL := /usr/bin/env bash
 .SUFFIXES:
 FORCE:
 
-# The version of approver-policy
-VERSION ?= $(shell git describe --tags --always --dirty --match='v*' --abbrev=14)
-
-BINDIR := $(CURDIR)/_bin
-
-.PHONY: all
-all: help
-
-ARCH   ?= $(shell go env GOARCH)
-OS     ?= $(shell go env GOOS)
-
-# Check https://github.com/helm/helm/releases for latest available release
-HELM_VERSION ?= 3.11.3
-# Check https://github.com/kubernetes-sigs/kubebuilder/blob/tools-releases/build/cloudbuild_tools.yaml
-# for latest available tag
-KUBEBUILDER_TOOLS_VERSION ?= 1.27.1
-# Check https://github.com/kyverno/kyverno/releases for latest available release
-KYVERNO_VERSION ?= v1.10.0
-K8S_CLUSTER_NAME ?= approver-policy
-IMAGE_REGISTRY ?= quay.io/jetstack
-IMAGE_NAME := cert-manager-approver-policy
-IMAGE_TAG := $(VERSION)
-IMAGE := $(IMAGE_REGISTRY)/$(IMAGE_NAME):$(IMAGE_TAG)
-
-GOMARKDOC_FLAGS=--format github --repository.url "https://github.com/cert-manager/approver-policy" --repository.default-branch master --repository.path /
-
-# An OCI Helm chart registry where the Helm package will be uploaded on release.
-# Empty by default to prevent accidental publication of the Helm chart.
-HELM_CHART_REGISTRY ?=
-
-helm_chart_archive := $(BINDIR)/charts/cert-manager-approver-policy-$(VERSION).tgz
-
-##@ General
-
-# The help target prints out all targets with their descriptions organized
-# beneath their categories. The categories are represented by '##@' and the
-# target descriptions by '##'. The awk commands is responsible for reading the
-# entire set of makefiles included in this invocation, looking for lines of the
-# file as xyz: ## something, and then pretty-format the target and help. Then,
-# if there's a line with ##@ something, that gets pretty-printed as a category.
-# More info on the usage of ANSI control characters for terminal formatting:
-# https://en.wikipedia.org/wiki/ANSI_escape_code#SGR_parameters
-# More info on the awk command:
-# http://linuxcommand.org/lc3_adv_awk.php
-
-.PHONY: help
-help: ## Display this help.
-	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
-
-##@ Utilities
-
-.PHONY: clean
-clean: ## clean up created files
-	rm -rf \
-		$(BINDIR) \
-		_artifacts
```

Comment:
`make clean` doesn't seem to be available any more.
How shall I delete all the generated files and start again if I need to?